.travis.yml

```diff
@@ -4,19 +4,19 @@ addons:
   apt:
     packages:
       - libgmp-dev
-      - openjdk-6-jre
+      - oracle-java8-installer
```

Comment:
Was this to diagnose/eliminate a runtime problem? I'd like to understand the motivation.